sessions/css-basics/social-card/index.html

```diff
@@ -1,56 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="utf-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-		<title>HTML & CSS</title>
-	</head>
-	<body>
-		<article>
-			<img src="./assets/github@2x.png" alt="a cartoon astronaut" />
-			<h2>GitHub</h2>
-			<p>See my cool Repos.</p>
-			<p>@myaccount</p>
-			<button>
-				<svg
-					xmlns="http://www.w3.org/2000/svg"
-					width="24"
-					height="24"
-					viewBox="0 0 24 24"
-					fill="none"
-					stroke="currentColor"
-					stroke-width="2"
-					stroke-linecap="round"
-					stroke-linejoin="round"
-				>
-					<circle cx="12" cy="12" r="10" />
-					<polyline points="12 16 16 12 12 8" />
-					<line x1="8" y1="12" x2="16" y2="12" />
-				</svg>
-			</button>
-		</article>
-		<article>
-			<img src="./assets/github@2x.png" alt="a cartoon astronaut" />
-			<h2>GitHub</h2>
-			<p>See my cool Repos.</p>
-			<p>@myaccount</p>
-			<button>
-				<svg
-					xmlns="http://www.w3.org/2000/svg"
-					width="24"
-					height="24"
-					viewBox="0 0 24 24"
-					fill="none"
-					stroke="currentColor"
-					stroke-width="2"
-					stroke-linecap="round"
-					stroke-linejoin="round"
-				>
-					<circle cx="12" cy="12" r="10" />
-					<polyline points="12 16 16 12 12 8" />
-					<line x1="8" y1="12" x2="16" y2="12" />
-				</svg>
-			</button>
-		</article>
-	</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>HTML & CSS</title>
+  </head>
+  <body>
+    <article>
+      <img src="./assets/github@2x.png" alt="a cartoon astronaut" />
```

Comment:
Since this image is merely decorative, I would leave the alt attribute empty.
```suggestion
      <img src="./assets/github@2x.png" alt="" />
```

Reply:
I did not write this exercise, but as far as I understood this image, it is meant to be a profile picture, which is why I would not set an empty alt-tag

Reply:
I guess it depends on whether any information the image holds is important for a screen reader user to know. And if the name of the user is elsewhere accessible.